src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory factory;
+
+    private HibernateUtil() {
+
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        if (factory == null) {
+            factory = new Configuration().configure().buildSessionFactory();
```

Comment:
You don't need to build it every time method is called, make the factory a singleton.